avm/res/recovery-services/vault/main.bicep

```
@@ -63,6 +63,15 @@ param securitySettings object = {}
 ])
 param publicNetworkAccess string = 'Disabled'
 
+@description('Optional. The encryption settings for the vault.')
+param encryption encryptionType
+
+@description('Optional. The redundancy settings of the vault.')
+param redundancySettings redundancySettingsType
+
+@description('Optional. The restore settings of the vault.')
+param restoreSettings restoreSettingsType
```

Just moving those up from below

```suggestion
param redundancySettings redundancySettingsType?

@description('Optional. The restore settings of the vault.')
param restoreSettings restoreSettingsType?
```

---

That being said, feel free to import the spec like shown here

https://github.com/Azure/bicep-registry-modules/blob/cb63d797073743d5abc1bd6a2fe73f66fdc531cc/avm/res/container-registry/registry/main.bicep#L141